cmd/version-control/main.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"net/http"
+	"strings"
+
+	"storj.io/storj/internal/version"
+)
+
+var (
+	ver []version.Info
+)
+
+func handleGet(w http.ResponseWriter, r *http.Request) {
+	var xfor string
+
+	// Only handle GET Requests
+	if r.Method == "GET" {
+		if xfor = r.Header.Get("X-Forwarded-For"); xfor == "" {
+			xfor = r.RemoteAddr
+		}
+		log.Printf("Request from: %s for %s", r.RemoteAddr, xfor)
+
+		w.Header().Set("Content-Type", "application/json")
+		response, err := json.Marshal(ver)
+		if err != nil {
+			w.WriteHeader(500)
```

Comment:
You should definitely return here and not continue writing to the writer.

Reply:
you can also do http.Error(w, err.Error(), 500) if you want to provide more info, but yeah you should return afterwards

Reply:
Damn it, thanks!
Totally forgot to return there